src/Adapter/SsiCache.php

```diff
@@ -42,23 +42,12 @@ class SsiCache implements CacheAdapterInterface
      */
     private $argumentResolver;
 
-    /**
-     * NEXT_MAJOR: make $argumentResolver mandatory when dropping sf < 3.1.
-     */
     public function __construct(
         string $token,
         RouterInterface $router,
         ?ControllerResolverInterface $resolver = null,
```

Comment:
Not a great idea to have a = null variable before one that cant be null

Reply:
Since this is master, I can change the places?